lib/popover.js

```diff
@@ -151,16 +151,16 @@ export const Popover = ({ align, renderLabel, views = {}, initialView, startOpen
   };
   // if a user has tabbed inside a popover and presses escape they should return back to the popover button
   const returnFocusOnClose = () => {
-    onClose()
+    onClose();
     toggleRef.current.focus();
-  }
+  };
 
   useEscape(open, returnFocusOnClose);
 
   // see https://www.w3.org/TR/wai-aria-practices-1.1/#menubutton
   return (
     <PopoverContainer data-module="Popover" ref={rootRef} {...props}>
-      {renderLabel({ onClick: onToggle, ref: toggleRef })}
+      {renderLabel({ onClick: onToggle, ref: toggleRef, open })}
```

Comment:
Should probably be `isOpen` to make clear that this is a bool, not an action (calling `open()` to try to open the popover would be an easy misconception)